### DISCUSSION TITLE
Authentication Failing

### DISCUSSION THREAD

**Discussion by VeryKross** [isaacrlevin/presencelight#934] [Q&A] [answered]
I'm curious if anyone else has just run into this. PL has been working great for over a year and yesterday it just stopped and it seems to have something to do with authentication to the graph. My account hasn't changed, in fact I haven't had to touch PL for most of a year now. I know of at least one other person at my office running PL who started experiencing this about the same time.

I'm hoping there's someone out there is a known fix/work-around as I've come to rely on using PL with my "BusyBuddy" gadget to keep my wife (who has mobility challenges) informed of when I'm attending online meetings in my home office at the other end of the house. It's amazing how much value that little light has provided 😊
https://github.com/VeryKross/BusyBuddy

**Comment by fdarrenlacy** [comment]
Was working yesterday and suddenly today cant log in.  I get an error message that the AAD application is disabled.  Did the app owner boink this?  One of my favorite hacks to sync my Hue bulbs all over the house with my corporate Teams status, broken as of today.

![2024-10-31_160956](https://github.com/user-attachments/assets/0973d9fa-e58c-41b5-b15c-cdf39afde390)

**Comment by VeryKross** [comment, marked as answer]
It appears that this was a change in our infrastructure/policy and we've been able to get it resolved with IT; the app is now authenticating properly again.
